.changeset/yellow-dots-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/virtualizer': patch
+---
+
+Changed the update behavior of virtualize directive to match lit-virtualizer's rendering of non-item changes.
```

Comment:
Suggest that we rephrase this to something more like:

The `virtualize` directive will now correctly re-render children when data stored outside the `items` array has changed.

Reply:
That reads much better.  Updated changeset.